database/connection.js

```diff
@@ -1,6 +1,6 @@
 const mongoose = require('mongoose');
 
-const mongoUri = 'mongodb://localhost/fec-photos';
+const mongoUri = 'mongodb://mongodb:27017/fec-photos';
 const db = mongoose.connect(mongoUri, {
```

Comment:
Was this the change that fixed everything?  So it tried to connect to the db container?

Reply:
Yes! @krisdo found the mistake in like 3 seconds.